core/services/job/orm.go

```diff
@@ -705,45 +700,20 @@ func (o *orm) FindJobs(offset, limit int) (jobs []Job, count int, err error) {
 			return err
 		}
 		for i := range jobs {
-			err = multierr.Combine(err, o.LoadEnvConfigVars(&jobs[i]))
+			o.LoadEnvConfigVars(&jobs[i])
 		}
 		return nil
 	})
 	return jobs, int(count), err
 }
 
-func (o *orm) LoadEnvConfigVars(jb *Job) error {
```

Comment:
AFAIK Its fine to remove  ocr and directRequest specs overriding because these are done in delegates anyway